handler_models.go

```diff
@@ -235,20 +235,19 @@ func ModelOwnerVersionZip(owner, name string, user *users.User, tx *gorm.DB,
 	if !valid {
 		modelVersion = ""
 	}
-	svc := &models.Service{Storage: globals.CloudStorage}
+	svc := &models.Service{Storage: globals.Storage}
 	_, zipPath, ver, em := svc.DownloadZip(r.Context(), tx,
 		owner, name, modelVersion, user, r.UserAgent())
 	if em != nil {
 		return nil, em
 	}
 
-	//zipFileName := fmt.Sprintf("model-%sv%d.zip", *model.UUID, ver)
-
-	// Remove request header to always serve fresh
-	//r.Header.Del("If-Modified-Since")
 	// Set zip response headers
+	zipFileName := fmt.Sprintf("%d.zip", ver)
 	w.Header().Set("Content-Type", "application/zip")
-	//w.Header().Set("Content-Disposition", fmt.Sprintf("attachment; filename=\"%s\"", zipFileName))
+	r.Header.Del("If-Modified-Since")
```

Comment:
Let's revert this change and move the header deletion back up where it was and include the comment explaining why we're doing this.